### PR TITLE
Add resources files and show them in the index page

### PR DESCRIPTION
**Pull request by smauro**
The idea is to have resources as yaml files and query them with graphql.

There is a simple example of showing them in the index page: normally, there would be another component to make the query and using the Card component.

### CODE REVIEW

File: src/pages/index.mdx [hunk 1]
```diff
@@ -3,9 +3,38 @@ title: Accueil
...
+      nodes {
+        link
+        description
+        author
+        name
+        pathology
+      }
+    }
+  }
+`
```
Comment (NatChe) [inline src/pages/index.mdx:40]: Probably move this to a separate component ? We can call it CategoryContent (or smth else) and pass the prop target to it. We should also think about including title and subtitle inside

File: src/resources/covid19-medicaments.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: Covid 19 Médicaments
+pathology: Information
+description: Test à effectuer en cas de symptomes et prise de traitement
+target: patient
+link: https://www.covid19-medicaments.com
+author: F. Le Goff
```
Comment (NatChe) [inline src/resources/covid19-medicaments.yaml:6]: Is there any particular reason to use yaml instead of json ? :)

**Comment by NatChe** [comment]
Bonne approche 👍! J'ai juste un doute sur le format yaml (je ne suis pas du tout fan ...) mais à discuter :)

File: src/resources/covid19-medicaments.yaml (hunk 2 above)
Comment (smauro) [reply src/resources/covid19-medicaments.yaml:6]: I thought about our discussion between json and markdown, and finally thought that yaml could be the good compromise between structured data and ease of use.
We can easily change between formats anytime, if we want.

File: src/pages/index.mdx (hunk 1 above)
Comment (smauro) [reply src/pages/index.mdx:40]: What are title and subtitle in this context?

File: src/resources/covid19-medicaments.yaml (hunk 2 above)
Comment (julienpasseri) [reply src/resources/covid19-medicaments.yaml:6]: yaml, no commas, no brackets. :-)

**Pull request merged**